build.gradle

```diff
@@ -11,6 +11,8 @@ plugins {
 
 apply from: 'common.gradle'
 
+applicationDefaultJvmArgs = ["-XstartOnFirstThread"]
```

Comment:
Looks like this line is great for Max OSX users, but doesn't work well for others.  Could we guard it so that it only takes effect on Mac OSX?